libs/utils/trace.py

```diff
@@ -81,7 +81,7 @@ def __init__(self, platform, data_dir, events,
                  plots_prefix=''):
 
         # The platform used to run the experiments
-        self.platform = platform
+        self.platform = platform or {}
```

Comment:
Yep, that's what this is trying to do (i.e. allow `Trace(None, path, events)` in which case we set `self.platform = {}`)